api/src/routing.js

```diff
@@ -51,8 +50,8 @@ var jsonParser = bodyParser.json({ limit: '32mb' });
 process
   .on('unhandledRejection', (reason, p) => {
     logger.error('UNHANDLED REJECTION!');
-    logger.error(reason)
-    logger.error(p)
+    logger.error(reason);
+    logger.error(p);
```

Comment:
p is type `Promise`? What does this log line look like ("[object Promise]")? Is it worthwhile to pull out information useful for the logs or even omit this?

Reply:
I've removed it altogether - I couldn't find any useful information on the promise.